Marlin/src/lcd/language/language_en.h

```diff
@@ -506,8 +506,8 @@ namespace Language_en {
   PROGMEM Language_Str MSG_KILL_EXPECTED_PRINTER           = _UxGT("INCORRECT PRINTER");
 
   #if LCD_WIDTH >= 20
-    PROGMEM Language_Str MSG_INFO_PRINT_COUNT              = _UxGT("Print Count");
-    PROGMEM Language_Str MSG_INFO_COMPLETED_PRINTS         = _UxGT("Completed");
+    PROGMEM Language_Str MSG_INFO_PRINT_COUNT              = _UxGT("Print Count: ");
+    PROGMEM Language_Str MSG_INFO_COMPLETED_PRINTS         = _UxGT("Completed: ");
     PROGMEM Language_Str MSG_INFO_PRINT_TIME               = _UxGT("Total Print Time");
     PROGMEM Language_Str MSG_INFO_PRINT_LONGEST            = _UxGT("Longest Job Time");
     PROGMEM Language_Str MSG_INFO_PRINT_FILAMENT           = _UxGT("Extruded Total");
```

Comment:
Do these not appear in the menu? Why do some need colons and some do not?

Reply:
Do some already have colons added in the code which uses the string, rather than here?

Reply:
These ones behave a little different.
Here a new line with ">" and the characters we wish to show is drawn. Thus not needing : from visual perspective.

Reply:
Tho if needed the lines i changed could be changed to behave like the ones you selected here in code.